state_machine/applications/system-check/tests/burnwire_test.py

```diff
@@ -0,0 +1,91 @@
+"""
+Python system check script for PyCubed satellite board
+PyCubed Mini mainboard-v02 for Pocketqube Mission
+Burnwire Deployment Test
+* Author(s): Yashika Batra
+"""
+
+from lib import pycubed as cubesat
+
+
+def get_user_input():
+    """
+    All user interaction happens in this function
+    Prompt users, get user inputs for voltage level and duration and return
+    """
+
+    # conduct the test
+    print("Initializing burnwire test...")
+    print("You can test using a multimeter or fishing line.")
+    print("If using a multimeter, please read the voltage between the" +
+          "ground pin (GND) and burnwire pin on the -Z solar board.")
+
+    # get user input for voltage level and duration
+    voltage = float(input(
+        "At what voltage do you want to run the burnwire IC? (up to 3.3V): "))
```

Comment:
Might be better to do say `(number between 0 and 3.3)` rather than `up to 3.3V` to make it clear what you're looking for.